libnavui-shield/src/main/java/com/mapbox/navigation/ui/shield/ShieldsCache.kt

```diff
@@ -217,7 +218,7 @@ internal class ShieldSpritesCache : ResourceCache<String, ShieldSprites>() {
     }
 }
 
-internal class ShieldByteArrayCache : ResourceCache<String, ByteArray>() {
+internal class ShieldByteArrayCache : ResourceCache<String, ByteArray>(3) {
```

Comment:
This should be a little bit smaller than `ShieldResultCache` since this caches background for multiple shields.

Reply:
I don't know if this should be a little smaller or the same size as `ShieldResultCache` because each shield background can be different. For ex: us-interstate-2 and us-interstate-3. I think we should take a safe side approach here and keep the size same.

Reply:
In testing across U.S., Japan and Europe I didn't observe a need to go above 10. I currently have it 15 but can adjust based on consensus.